Sources/ComposedUI/CollectionView/UICollectionViewSection.swift

```diff
@@ -0,0 +1,45 @@
+import UIKit
+import Composed
+
+@available(*, deprecated, renamed: "SingleUICollectionViewSection")
+public typealias CollectionSectionProvider = SingleUICollectionViewSection
```

Comment:
Hmm interesting naming?

Reply:
I'm not sure what the naming here should be. It's trying to advertise that it's a `Section` that when in a `UICollectionView` provides a single type of cell.

I believe the requirements haven't changes from `CollectionSectionProvider`, but it's to differentiate it from `UICollectionViewSection` which allows for different cell types per-row (rather than the current per-section).

So it's safe to say: a lot of these names are placeholders for now 😅